contrib/net/http/http_test.go

```diff
@@ -109,10 +109,20 @@ func TestAnalyticsSettings(t *testing.T) {
 		w := httptest.NewRecorder()
 		mux.ServeHTTP(w, r)
 
+		f := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			message := "Hello \n"
```

Comment:
Can you please create a separate test? Let’s aim for small tests, instead of sneaking in some assertion into a helper function. Seems very out of place.

Reply:
👍 We want to test all these test cases on WrapHandler. This is a gap in our testing. I've put them into their own test cases so one test tests one thing.